### ISSUE TITLE
error: Watchman:  spawn watchman ENOENT

### ISSUE THREAD

**Issue by lervag** [neoclide/coc.nvim#1975]
## Result from CocInfo

```
vim version: NVIM v0.5.0-132-g83b6d9f19
node version: v14.2.0
coc.nvim version: 0.0.78-05f8f7bd71
term: xterm
platform: linux
```

## Describe the bug

When I start neovim after recent update, I get the following error:

```
[coc.nvim] error: Watchman:  spawn watchman ENOENT
[coc.nvim] error: Uncaught exception: Error: spawn watchman ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:268:19)
    at onErrorNT (internal/child_process.js:468:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
```

I don't know much javascript, but it seems this is due to `watchman` not existing. However, I never got this error before (not even earlier today). If necessary, I'll be happy to construct a minimal example.

**Comment by chemzqm** [comment]
Fixed by f552d4d7

**Comment by lervag** [comment]
Thanks!
